config.py

```
@@ -1,73 +1,92 @@
 
-    def ReadConfig(self, configName):
+    def read_config(self, config_name):
         config = configparser.ConfigParser()
-        config.read(configName)
+        config.read(config_name)
 
         try:
             self.url = config.get("Settings", "url")
         except:
-            self.url = 'https://www.fusker.xxx/en/'
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)